src/nodes/html.ts

```diff
@@ -421,40 +421,43 @@ export default class HTMLElement extends Node {
 			return this._rawAttrs;
 		const attrs = {} as RawAttributes;
 		if (this.rawAttrs) {
-			const re = /\b([a-z][a-z0-9\-]*)(?:\s*=\s*("(?:[^"]*)"|'(?:[^']*)'|(?:\S+)))?/ig;
+			const re = /\b([a-z][a-z0-9\-]*)(?:\s*=\s*(?:"([^"]*)"|'([^']*)'|(\S+)))?/ig;
 			let match: RegExpExecArray;
 			while (match = re.exec(this.rawAttrs)) {
-				attrs[match[1]] = match[2] || null;
+				attrs[match[1]] = match[2] || match[3] || match[4] || null;
 			}
 		}
 		this._rawAttrs = attrs;
 		return attrs;
 	}
 
+	/**
+	 * Get an attribute
+	 * @return {string} value of the attribute
+	 */
+	getAttribute(key: string) {
+		return this.attributes[key] || null;
+	}
+
 	/**
 	 * Set an attribute value to the HTMLElement
 	 * @param {string} key The attribute name
 	 * @param {string|number} value The value to set, or null / undefined to remove an attribute
 	 */
 	setAttribute(key: string, value: string | number) {
-		// Update the this.attributes
+		// Invalidate current this.attributes
 		if (this._attrs) {
 			delete this._attrs;
 		}
-		const attrs = this.rawAttributes;	// ref this._rawAttrs
```

Comment:
Why this comment is removed? I put this here just to remind me.

Reply:
I removed this comment, because it's misleading, I read it like reference to `_rawAttrs`, without `rawAttributes` if you do `_rawAttrs`, it will return `undefined`. `_rawAttrs` simply cached the result of `rawAttributes` to speed it up.